pkg/common/cns-lib/volume/manager.go

```diff
@@ -155,8 +155,13 @@ func (m *volumeManager) CreateVolume(spec *cnstypes.CnsVolumeCreateSpec) (*cnsty
 			klog.Errorf("Failed to get taskInfo for CreateVolume task from vCenter %q with err: %v", m.virtualCenter.Config.Host, err)
 			return nil, err
 		}
-		// Store the taskInfo details and taskInfo object expiration time in volumeTaskMap
-		volumeTaskMap[spec.Name] = createVolumeTaskDetails{taskInfo, time.Now().Add(time.Hour * time.Duration(defaultOpsExpirationTimeInHours))}
+		blockBackingDetails, ok := spec.BackingObjectDetails.(*cnstypes.CnsBlockBackingDetails)
```

Comment:
Can there be a case where `BackingObjectDetails` is not of type `cnstypes.CnsBlockBackingDetails` and we get `!ok`?
In that case, we will not be caching the task.

Reply:
> Can there be a case where BackingObjectDetails is not of type cnstypes.CnsBlockBackingDetails and we get !ok?
No. 
It can be fileType only for 2.0 and beyond. This PR is only for 1.0

Reply:
Refer: https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/545